kubernetes/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for torchserve helm chart.
+
+torchserve_image: pytorch/torchserve:latest-gpu
+
+namespace: torchserve
+
+torchserve:
+  management_port: 8081
+  inference_port: 8080
+  pvd_mount: /home/model-server/shared/
+  n_gpu: 1
+  n_cpu: 1
+  memory_limit: 4Gi
+  memory_request: 1Gi
+
+deployment:
+  replicas: 1
+
+persitant_volume:
+  size: 1Gi
```

Comment:
@dhanainme Where is this size getting used? If user's have lots of models they will need to increase. Add some description in readme for this

Reply:
Have added a note for templates/efs_pv_claim.yaml & this.